### PR TITLE
Interop Client CLI Improvements

### PR DESCRIPTION
**Pull request by holgerd77**
Some interop related CLI Improvements for the client, WIP.

### CODE REVIEW

File: packages/client/lib/client.ts [hunk 1]
```diff
@@ -120,6 +122,14 @@ export class EthereumClient {
 
     await Promise.all(this.services.map((s) => s.open()))
 
+    const ts = this.chain.genesis.header.timestamp
+    const tsDiff = Number(ts) - Date.now()
```
Comment (jochem-brouwer) [inline packages/client/lib/client.ts:126]:
```suggestion
    const tsDiff = Number(ts) - Date.now()/1000
```
`Date.now() is in milliseconds`

**Review by jochem-brouwer — COMMENTED** [review]
One suggestion otherwise LGTM

**Review by jochem-brouwer — DISMISSED** [review]
LGTM

**Review by g11tech — APPROVED** [review]
lgtm!

**Review by jochem-brouwer — APPROVED** [review]

**Pull request merged**